kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -580,6 +580,17 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  val FRONTEND_PROXY_HTTP_CLIENT_IP_HEADER: ConfigEntry[String] =
```

Comment:
why we need this conf?

Reply:
It is a config for the proxy.

I am using ebay hlb, the header is `X-eBay-Web-Tier-IP`.